# This is a BitKeeper generated diff -Nru style patch.
#
# ChangeSet
#   2005/01/03 20:14:02-08:00 rjmx@rjmx.net 
#   [PATCH] CS461x gameport code isn't being included in build
#   
#   With Cal Peake <cp@absolutedigital.net>
#   
#   I've found a typo in drivers/input/gameport/Makefile in kernel 2.6.9 which
#   effectively prevents the CS461x gameport code from being included.
#   
#   Signed-off-by: Ron Murray <rjmx@rjmx.net>
#   Signed-off-by: Cal Peake <cp@absolutedigital.net>
#   Signed-off-by: Andrew Morton <akpm@osdl.org>
#   Signed-off-by: Linus Torvalds <torvalds@osdl.org>
# 
# drivers/input/gameport/Makefile
#   2005/01/03 15:49:13-08:00 rjmx@rjmx.net +1 -1
#   CS461x gameport code isn't being included in build
# 
diff -Nru a/drivers/input/gameport/Makefile b/drivers/input/gameport/Makefile
--- a/drivers/input/gameport/Makefile	2005-01-04 18:39:49 -08:00
+++ b/drivers/input/gameport/Makefile	2005-01-04 18:39:49 -08:00
@@ -5,7 +5,7 @@
 # Each configuration option enables a list of files.
 
 obj-$(CONFIG_GAMEPORT)		+= gameport.o
-obj-$(CONFIG_GAMEPORT_CS461X)	+= cs461x.o
+obj-$(CONFIG_GAMEPORT_CS461x)	+= cs461x.o
 obj-$(CONFIG_GAMEPORT_EMU10K1)	+= emu10k1-gp.o
 obj-$(CONFIG_GAMEPORT_FM801)	+= fm801-gp.o
 obj-$(CONFIG_GAMEPORT_L4)	+= lightning.o
